go.mod

```diff
@@ -23,6 +23,7 @@ require (
 	github.com/mattbaird/jsonpatch v0.0.0-20171005235357-81af80346b1a // indirect
```

Comment:
Why no deps are removed, shoudn't at least knative deps get removed?

Reply:
You're right. I should have double checked go.mod file. I was quite sure that they would be removed automatically from go.mod. I'll remove them.

Reply:
Fixed them, see commit 577c562e5c2def8fb5d9c8a04b906f1ed59ba857.